test/e2e_node/inode_eviction_test.go

```diff
@@ -342,3 +318,32 @@ func hasInodePressure(f *framework.Framework, testCondition string) (bool, error
 	}
 	return hasPressure, nil
 }
+
+// returns a pod that does not use any resources
+func getInnocentPod() *v1.Pod {
+	return &v1.Pod{
+		ObjectMeta: metav1.ObjectMeta{Name: "innocent-pod"},
+		Spec: v1.PodSpec{
+			RestartPolicy: v1.RestartPolicyNever,
+			Containers: []v1.Container{
+				{
+					Image: "gcr.io/google_containers/busybox:1.24",
+					Name:  "innocent-container",
+					Command: []string{
+						"sh",
+						"-c",
+						"while true; do sleep 5; done",
+					},
+				},
+			},
+		},
+	}
+}
+
+func getInodeConsumingCommand(path string) []string {
+	return []string{
+		"sh",
+		"-c",
+		fmt.Sprintf("i=0; while true; do touch %ssmallfile$i.txt; sleep 0.001; i=$((i+=1)); done;", path),
```

Comment:
Suggest `fmt.Sprintf("...%s${i}.txt;...", filepath.Join(path, "smallfile")`, so you don't have to worry about including a trailing slash on the `path` argument.